test/interceptors/response.test.ts

```diff
@@ -100,6 +100,16 @@ describe('test response interceptor', () => {
     expect(resultCache.cached).toBe(true);
   });
 
+  it('tests header interpreter integration deprecated', async () => {
```

Comment:
Move this to the header interpreter tests file.

```suggestion
  it('tests header interpreter returning only numbers', async () => {
```